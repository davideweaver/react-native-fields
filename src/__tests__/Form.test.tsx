import * as React from 'react';
import * as renderer from 'react-test-renderer';

import { Form } from '..';

describe('HomeScreen', () => {

    it('should match basic snapshot', async () => {

        const component = renderer.create(
            <Form />,
        );

        expect(component.toJSON()).toMatchSnapshot();
    });
});
