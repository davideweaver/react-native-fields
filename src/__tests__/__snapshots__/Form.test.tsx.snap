// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen should match basic snapshot 1`] = `<View />`;
